app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(players: list):
+    return sum(player.get_rating() for player in players)
+
```

Comment:
Add type annotation for return value in main.py

Reply:
Thx, fixed :)